nautobot/extras/signals.py

```diff
@@ -146,6 +191,10 @@ def _handle_changed_object(sender, instance, raw=False, **kwargs):
     if raw:
         return
 
+    if kwargs.get("signal") == pre_save and not kwargs.get("created"):
+        _ensure_changelog_exists(ObjectChangeActionChoices.ACTION_UPDATE, sender, instance)
+        return
```

Comment:
Given the code flow for `pre_save` is completely distinct from the code flow for `post_save` and `m2m_changed`, I'd prefer that this be a separate handler function rather than piggybacking on the existing one.